modules/serverless-ossm-secret-filtering-net-kourier.adoc

```diff
@@ -42,3 +40,23 @@ spec:
          value: 'true'
      name: net-kourier-controller
 ----
+
+You can disable the secret filtering by setting `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=false` via `workloads` in the `KnativeServing` custom resource (CR).
+
+.Example for `net-kourier`
+[source,yaml]
+-----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+ name: knative-serving
+ namespace: knative-serving
+spec:
+ workloads:
+   - env:
+     - container: controller
+       envVars:
+       - name: ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID
+         value: 'false'
+     name: net-kourier-controller
```

Comment:
One last nit, we should probably fix indentation in the new examples to 2 spaces, like this:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
metadata:
  name: knative-serving
  namespace: knative-serving
spec:
  workloads:
    - env:
        - container: controller
          envVars:
            - name: ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID
              value: 'false'
      name: net-kourier-controller
```

Reply:
Okay! I will update it. Can we use the merged example for `net-kourier` as well? Just like you provided for the `net-istio`? If yes, can you please provide me one so that I am not missing anything.

Reply:
For net-kourier we just want to remove the first one (there's nothing to merge) and keep the second one, with fixed indentation like above.

Reply:
Okay, noted!

Reply:
Yeah I think that either you omit the first one or have something like:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
metadata:
 name: knative-serving
 namespace: knative-serving
```
So you start with a simple CR and you add the env var in the second one.